### ISSUE TITLE
FileNotFoundError while running the code

### ISSUE THREAD

**Issue by Dhawaleswrar** [cheng6076/NeuralSum#5]
While trying to run "pretrain.py and train.py" files in Ubuntu 16.04, facing the following problem.

reading train
Traceback (most recent call last):
  File "train.py", line 343, in <module>
    tf.app.run()
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/platform/app.py", line 124, in run
    _sys.exit(main(argv))
  File "train.py", line 209, in main
    load_data(FLAGS.data_dir, FLAGS.max_doc_length, FLAGS.max_sen_length)
  File "/home/sks/1 project/NeuralSum-master/data_reader.py", line 72, in load_data
    for dname in os.listdir(pname):  
FileNotFoundError: [Errno 2] No such file or directory: 'data/demo/train'

**Comment by JafferWilson** [comment]
@Dhawaleswrar  I guess you should read the Read.me file. There you can get the link for the data.

**Comment by Dhawaleswrar** [comment]
Yes Sir, I already downloaded and this is the error coming after downloading 'neuralsum' folder from data link and project from 'https://github.com/cheng6076/NeuralSum.git'.

**Comment by JafferWilson** [comment]
Had you gone through the process of pretrain after you downloaded the data? Please let me know the process you have tried running the repository in detail.

**Comment by Dhawaleswrar** [comment]
Sir, my team has followed the procedure written in pretrain.py and train.py to run the repo, but unable to run it and each one of us facing different errors. Kindly, tell us the procedure how to configure and run the project in detail?

**Comment by JafferWilson** [comment]
What different errors you are talking about? Mention them here if possible..... Sorry but thats the only procedure which is written in the READ.ME file.

**Comment by npochhi** [comment]
@JafferWilson Where should we place the downloaded data?
